AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -43,16 +40,57 @@ export var functionsFlow = {
 
         if (!await isKuduAccessiblePromise)
         {
-            if (siteInfo.kind.includes("linux") || siteInfo.kind.includes("container")){
-                flowMgr.addView(new CommonWordings().connectivityCheckUnsupported.get());
-            }else{
-                flowMgr.addView(new VnetDnsWordings().cannotCheckWithoutKudu.get("Functions settings"));
-            }
+            flowMgr.addView(new VnetDnsWordings().cannotCheckWithoutKudu.get("Functions settings"));
             return;
         }
-
-        var isDaasExtAccessible = await diagProvider.checkDaasExtReachable(30);
-
+        var param = [];

```

Comment:
Please move the entire daas version determination logic to a separate function in this file to improve readability.  This function is already very long.

Reply:
Having this function in flowMisc.js is ideal as it could be used by the web app flow as well.

Reply:
fixed